src/app/contextBroker/contextBroker.cpp

```diff
@@ -1574,6 +1574,9 @@ static void notificationModeParse(char *notifModeArg, int *pQueueSize, int *pNum
 int main(int argC, char* argV[])
 {
   strncpy(transactionId, "N/A", sizeof(transactionId));
+  strncpy(srv,           "N/A", sizeof(srv));
+  strncpy(subsrv,        "N/A", sizeof(subsrv));
+  strncpy(from,          "N/A", sizeof(from));
 
```

Comment:
It would be better to encapsulate the new LM-vars.
E.g.:
  lmServiceSet("N/A");
  lmSubServiceSet("N/A");
  lmClientIpSet("N/A");

Not sure what to call 'from'. 'from' is definitely not good enough.
clientIp, peerIp ... ?